cluster-autoscaler/vendor/github.com/gardener/machine-controller-manager/pkg/apis/machine/types.go

```diff
@@ -946,6 +946,7 @@ type AzureVirtualMachineProperties struct {
 	OsProfile       AzureOSProfile
 	NetworkProfile  AzureNetworkProfile
 	AvailabilitySet AzureSubResource
+	Zone            *int
```

Comment:
This is a workaround for now. 
We should ideally re-vendor the MCM, but it seems challenging due to recently updated Azure-dependencies in MCM, I would prefer to do it separately.